gsa/src/web/components/link/certlink.js

```diff
@@ -25,7 +25,12 @@ import PropTypes from 'web/utils/proptypes';
 import DetailsLink from './detailslink';
 
 const CertLink = ({id, textOnly = false, type}) => {
-  if (type !== 'CERT-Bund' && type !== 'DFN-CERT') {
+  if (
+    type !== 'cert-bund' &&
+    type !== 'CERT-Bund' &&
+    type !== 'dfn-cert' &&
+    type !== 'DFN-CERT'
```

Comment:
Why are both writing (lower and upper case) possible. We should really avoid such things. If we really need this, it would also be better to user type.toLowerCase() instead